tests/wicked/advanced/sut/t19_team_interface_legacy.pm

```diff
@@ -29,10 +29,11 @@ sub run {
 
     record_info('Info', 'Create a team interface from legacy ifcfg files');
 
-    $self->get_from_data('wicked/teaming/ifcfg-eth0',  $cfg_ifc0);
-    $self->get_from_data('wicked/teaming/ifcfg-eth1',  $cfg_ifc1);
-    $self->get_from_data('wicked/teaming/ifcfg-team0', $cfg_team0);
-    file_content_replace($cfg_team0, ipaddr4 => $self->get_ip(type => 'host', netmask => 1), ipaddr6 => $self->get_ip(type => 'host6', netmask => 1), port0 => $ctx->iface(), port1 => $ctx->iface2());
+    $self->get_from_data('wicked/ifcfg/noip_hotplug', $cfg_ifc0);
+    $self->get_from_data('wicked/ifcfg/noip_hotplug', $cfg_ifc1);
+    $self->get_from_data('wicked/ifcfg/team0',        $cfg_team0);
+    file_content_replace($cfg_team0, ipaddr4 => $self->get_ip(type => 'host', netmask => 1), ipaddr6 => $self->get_ip(type => 'host6', netmask => 1), iface0 => $ctx->iface(), iface1 => $ctx->iface2());
+    zypper_call('-q in libteam-tools libteamdctl0 python-libteam');
```

Comment:
I will move this to before_test, as it's also needed in other tests...

Reply:
agree , feel free to do it in your branch . Or I will do it in another one since this one is closed